ivadomed/main.py

```diff
@@ -836,8 +795,10 @@ def cmd_eval(context):
         subj, acq = subj_acq.split('_')[0], '_'.join(subj_acq.split('_')[1:])
 
         fname_pred = os.path.join(path_pred, subj_acq + '_pred.nii.gz')
-        fname_gt = os.path.join(context['bids_path'], 'derivatives', 'labels', subj, 'anat',
-                                subj_acq + context['target_suffix'] + '.nii.gz')
+        fname_gt = []
+        for suffix in context['target_suffix']:
+            fname_gt.append(os.path.join(context['bids_path'], 'derivatives', 'labels', subj, 'anat',
+                                         subj_acq + suffix + '.nii.gz'))
 
         # 3D evaluation
         eval = Evaluation3DMetrics(fname_pred=fname_pred,
```

Comment:
in cmd_test why return metrics_dict? The variable is not always assigned a this point causing an error

Reply:
I agree!
I think it has been added for this application: https://github.com/neuropoly/ivado-medical-imaging/blob/83a7b58d8458393b04dbe2f9c876221d95471db6/dev/automate_training.py#L88